ARK Server Manager/Lib/Model/ServerProfile.cs

```diff
@@ -1889,5 +1896,25 @@ public void ResetOverrideMaxExperiencePointsDino()
         {
             OverrideMaxExperiencePointsDino = GameData.DEFAULT_MAX_EXPERIENCE_POINTS_DINO;
         }
+
+        // section reset methods
+        public void ResetChatAndNotificationSection()
+        {
+            EnableGlobalVoiceChat = GameData.DEFAULT_ENABLE_GLOBAL_VOICE_CHAT;
```

Comment:
Alternatively, we could not use the default values in the properties and instead have a function which calls all of the reset functions.